docs/bcd-workflow.md

```diff
@@ -0,0 +1,34 @@
+# BCD workflow
+
+The [web-platform-dx/web-features](https://github.com/web-platform-dx/web-features/) and the [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data/) depend on and complement each other. This document describes the workflows which synchronize the two projects.
+
+## BCD keys in `compat_features`
+
+In web-feature YAML files, the `compat_features` field consists of @mdn/browser-compat-data (BCD) entry keys (e.g., `css.properties.background-color`) that make up this feature.
+
+If `compat_features` is not set in `<feature-id>.yml`, the `dist` script will populate `compat_features` in `<feature-id>.yml.dist` with BCD entry keys tagged with `web-features:<feature-id>` in BCD, if any exist.
+
+If a `compat_features` list is in `<feature-id>.yml`, it takes precedence over BCD tags.
+
+## Tagging keys in BCD directly
+
+In this workflow, new keys in BCD are tagged from inception. This is more likely for small, incremental changes to existing features.
```

Comment:
```suggestion
This way of defining which BCD keys make up a feature happens in BCD itself. This way is more suited for small, incremental changes to existing features.
```

I was a big confused by the word "likely". Are we saying that's what is happening now, or what we want to happen? If we're merely describing what's happening now, then "likely" sort of makes more sense. But if we are offering guidelines for how to do things, I think "suited" is the better choice.

Reply:
Edit: I now realize that the word "workflow" might be intentional, because this talks about github workflow that runs automatically. If it is the case, it wasn't clear to me, so I suggest making this more obvious in the text.